src/instruments/src/ISIS/ArtificialHorizonDisplay.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { useInteractionEvent } from '@instruments/common/hooks';
+import { useSimVar } from '@instruments/common/simVars';
+
+import { AltitudeIndicator } from './AltitudeIndicator';
+import { LandingSystem } from './LandingSystem';
+import { MachIndicator } from './MachIndicator';
+import { PressureIndicator } from './PressureIndicator';
+import { ArtificialHorizon } from './ArtificialHorizon';
+import { AirspeedIndicator } from './AirspeedIndicator';
+import { AirplaneSymbol } from './AirplaneSymbol';
+import { Bug, BugType } from './Bug';
+
+type Props = {
+    indicatedAirspeed: number;
+    bugs: Bug[];
+}
+
+export const ArtificialHorizonDisplay: React.FC<Props> = ({ indicatedAirspeed, bugs }) => {
+    const [alt] = useSimVar('INDICATED ALTITUDE:2', 'feet');
+    const [mda] = useSimVar('L:AIRLINER_MINIMUM_DESCENT_ALTITUDE', 'feet');
+
+    const [lsActive, setLsActive] = useState(false);
+    useInteractionEvent('A32NX_ISIS_LS_PRESSED', () => setLsActive(!lsActive));
```

Comment:
Let's use an LVar here. Some external hardware / apps might want to read into this state.

Reply:
Good point, I am guessing I would use the `useInteractionSimVar` hook for this?

I think I should also be serializing all the bugs to LVars, right? What kept me from doing it was not knowing where to put the initialization logic. Should I just do a `useEffect(..., [])` to check if the vars exist and set them to the default values (30 for speed, 100 for alt) if they don't? Didn't feel great putting all of this initialization logic into the component. 
Or should I put them into the `apron.flt`, `runway.flt`, etc.? I think they would have the same initial values regardless of spawning position, so that felt kinda wrong too.

I was thinking of using `L:A32NX_ISIS_BUGS_ALT_VALUE:i` (i = 0,1) and `L:A32NX_ISIS_BUGS_SPD_VALUE:j` (i = 0, 1, 2, 3) to store the 6 values and either using `L:A32NX_ISIS_BUGS_ALT_ACTIVE:i` and `L:A32NX_ISIS_BUGS_SPD_ACTIVE:j` or just using a single integer variable and encoding the active status with bit flags which would save some space but make it a little less intuitive for someone trying to use them.